test/common/fit_common.py

```diff
@@ -473,9 +471,13 @@ def remote_shell(shell_cmd, expect_receive="", expect_send="", timeout=300,
 
 
 def scp_file_to_ora(src_file_name):
+    # legacy call
```

Comment:
Is there any reason not to just update these all? There are only a handful in 3 files:
test/deploy/rackhd_package_install.py:        fit_common.scp_file_to_ora("sudoersproxy")
test/deploy/rackhd_package_install.py:        fit_common.scp_file_to_ora('control.cfg')
test/deploy/rackhd_package_install.py:            fit_common.scp_file_to_ora('pdudirect.cfg')
test/deploy/rackhd_package_install.py:        fit_common.scp_file_to_ora('dhcpd.conf')
test/deploy/rackhd_package_install.py:        fit_common.scp_file_to_ora('config.json')
test/deploy/rackhd_package_install.py:        fit_common.scp_file_to_ora('rabbitmq.config')
test/deploy/rackhd_source_install.py:    fit_common.scp_file_to_ora('settings.xml')
test/deploy/rackhd_source_install.py:        fit_common.scp_file_to_ora("sudoersproxy")
test/deploy/rackhd_source_install.py:        fit_common.scp_file_to_ora('control.cfg')
test/deploy/rackhd_source_install.py:            fit_common.scp_file_to_ora('pdudirect.cfg')
test/deploy/rackhd_source_install.py:        fit_common.scp_file_to_ora('dhcpd.conf')
test/deploy/rackhd_source_install.py:        fit_common.scp_file_to_ora('config.json')
test/deploy/rackhd_source_install.py:        fit_common.scp_file_to_ora('rabbitmq.config')
test/deploy/rackhd_stack_init.py:        fit_common.scp_file_to_ora('auth.json')